utils/minicom/Makefile

```diff
@@ -11,8 +11,8 @@ PKG_NAME:=minicom
 PKG_VERSION:=2.7
 PKG_RELEASE:=2
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://alioth.debian.org/frs/download.php/file/3977/
+PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).orig.tar.gz
+PKG_SOURCE_URL:=http://ftp.debian.org/debian/pool/main/m/minicom/
```

Comment:
No, we're not going to use some non standard filename and main site works just fine.
The mirrors listed here are all fine and hosts minicom 2.7
https://github.com/lede-project/source/pull/1133/commits/694dffa37121610667a922520b9505493272b833

Reply:
Thank you for your feedback, I'm quite new with openwrt codebase ¿What do you suggest?
¿Grabbing some of those mirrors and setting them as PKG_SOURCE or adding that patch to download.pl script (not sure how that exactly works)?

Reply:
I would love to have that patch in the tree as it would be an easy way to provide mirrors and reliable ones. If more people wants this in it'll most likely change the core devs minds. Please make a comment why you'd like to see this in.